config/multiband.py

```diff
@@ -0,0 +1,11 @@
+# overrides for pipe_tasks MultiBandTask
+
+import os.path
+
+from lsst.utils import getPackageDir
+
+for sub in ("detectCoaddSources", "mergeCoaddDetections", "measureCoaddSources", "mergeCoaddMeasurements",
+            "forcedPhotCoadd"):
+    path = os.path.join(getPackageDir("obs_cfht"), "config", sub + ".py")
+    if os.path.exists(path):
+        getattr(root, sub).load(path)
```

Comment:
I don't think we need this config file (we do need the other two), at least not yet - it's for the multiBand.py driver we're not porting over yet, because it relies on hscPipeBase features we're not porting over yet.